src/shared/parameter_v2/config_definitions/simulated_test_main_command_line_args.yaml

```diff
@@ -4,6 +4,12 @@
     description: >-
         Displays simulated test on visualizer
 
+- bool:
+    name: stop_ai_on_start
```

Comment:
Not really an issue, but what if the flag was just --stop_ai? Less typing and its just as descriptive.

Reply:
Hm, that seems confusing to me, because it sounds like we're running a simulated test with a stopped AI

Reply:
yeah sorry, I see it now